airbyte-webapp/src/packages/cloud/views/credits/CreditsPage/CreditsPage.tsx

```diff
@@ -65,25 +32,7 @@ const CreditsPage: React.FC = () => {
       <Content>
         {!emailVerified && <EmailVerificationHintWithMargin />}
         <RemainingCredits selfServiceCheckoutEnabled={emailVerified} />
-        <MainInfo>
-          <SideMenu data={menuItems} onSelect={onSelectMenuItem} activeItem={pathname} />
-          <MainView>
-            <Suspense fallback={<LoadingPage />}>
-              <Routes>
-                {menuItems.flatMap((menuItem) =>
-                  menuItem.routes.map(({ path, component: Component }) => (
-                    <Route key={`${path}`} path={`${path}`} element={<Component />} />
-                  ))
-                )}
-
-                <Route
-                  path="*"
-                  element={<Navigate to={firstRoute ? `${menuItems?.[0].routes?.[0]?.path}` : CloudRoutes.Root} />}
-                />
-              </Routes>
-            </Suspense>
-          </MainView>
-        </MainInfo>
+        <CreditsUsagePage />
```

Comment:
nit: should this component be renamed to something like `CreditsUsage`, since it is no longer being considered as a "separate page", but instead just a part of the CreditsPage?

Reply:
@lmossman That's a really good point, thank you!